src/libraries/System.Runtime.Extensions/tests/System/Environment.Exit.cs

```diff
@@ -41,7 +41,7 @@ public static void ExitCode_Roundtrips(int exitCode)
         [InlineData(1)] // setting ExitCode and exiting Main
         [InlineData(2)] // setting ExitCode both from Main and from an Unloading event handler.
         [InlineData(3)] // using Exit(exitCode)
-        [SkipOnPlatform(TestPlatforms.Android | TestPlatforms.Browser | TestPlatforms.iOS | TestPlatforms.MacCatalyst | TestPlatforms.tvOS, "Not supported on Android, Browser, iOS, MacCatalyst, or tvOS.")]
+        [SkipOnPlatform(TestPlatforms.Android | TestPlatforms.Browser | TestPlatforms.iOS | TestPlatforms.MacCatalyst | TestPlatforms.tvOS, "Not supported on Android, Browser, iOS, MacCatalyst, or tvOS. External processes cannot be launched.")]
```

Comment:
Should this rather use `ConditionalTheory` with `nameof(RemoteExecutor.IsSupported)`?

Reply:
Good catch! Updated.